nengo_loihi/probes.py

```diff
@@ -13,3 +13,27 @@ def __init__(self, target=None, key=None, slice=None, weights=None,
 
     def validate(self):
         pass
+
+
+class ProbeGroup(object):
+    def __init__(self):
+        self.probes = []
+
+    def __iter__(self):
+        return iter(self.probes)
+
+    def add(self, probe):
+        """Add a Probe object to ensemble."""
+        # if probe.target is None:
+        #     probe.target = self
+        # assert probe.target is self
```

Comment:
Leftover comments

Reply:
So, I'm silly. Of course the assert doesn't work anymore, because the probe targets the NeuronGroup, not this ProbeGroup. To check that it targets the NeuronGroup, we'd have to pass that in to the ProbeGroup and store a handle there. Maybe checking this is too much hand-holding. There's lots of ways that someone could mess up when trying to put together a model, and we can't catch all of them. Also, this would be double-checking for devs, not front-end users, since all these objects are configured by the builder for front-end users.

The other option (and I wouldn't do this here, due to time requirements), is if Probes are always targeting the NeuronGroup that contains them, then we don't need a target parameter at all, we can just infer it when we build the probe.

Reply:
Yeah I'm fine leaving the assertion out since it's only dev-facing.